orbit/tools/cleanup/cleanup_macos.sh

```diff
@@ -5,3 +5,5 @@ sudo launchctl unload /Library/LaunchDaemons/com.fleetdm.orbit.plist
 
 sudo pkill fleet-desktop || true
 sudo rm -rf /Library/LaunchDaemons/com.fleetdm.orbit.plist /var/lib/orbit /usr/local/bin/orbit /var/log/orbit /opt/orbit/
+
+sudo pkgutil --forget com.fleetdm.orbit.base.pkg
```

Comment:
Looks good!

Just verified and it indeed gets rid of `/var/db/receipts/com.fleetdm.orbit.base.pkg.bom` and `/var/db/receipts/com.fleetdm.orbit.base.pkg`.

One small thing: Could you add a ` || true` at the end of the command? (so that the command and script doesn't error out if there's nothing to cleanup).